runtime/port/win32/j9shmem.c

```diff
@@ -90,6 +90,10 @@ j9shmem_open(struct J9PortLibrary *portLibrary, const char* cacheDirName, uintpt
 		Trc_PRT_shmem_j9shmem_open_ExitNullCacheDirName();
 		return J9PORT_ERROR_SHMEM_OPFAILED;
 	}
+	
+	if (NULL != controlFileStatus) {
+		memset(controlFileStatus, 0, sizeof(J9ControlFileStatus));
```

Comment:
Technically, this should be done before the `return J9PORT_ERROR_SHMEM_OPFAILED` above.

Reply:
Updated.

Reply:
We should update Unix as well, but that can be done in a separate PR.

Reply:
> We should update Unix as well, but that can be done in a separate PR.

PR: https://github.com/eclipse/openj9/pull/12003